main/src/ui/add_conversation/list_row.vala

```diff
@@ -8,27 +8,55 @@ namespace Dino.Ui {
 
 public class ListRow : Widget {
 
-    public Grid outer_grid;
+    public Box outer_box;
     public AvatarPicture picture;
     public Label name_label;
+    public Image status_dot;
     public Label via_label;
 
+    public string? status_str;
     public Jid? jid;
     public Account? account;
 
     construct {
         Builder builder = new Builder.from_resource("/im/dino/Dino/add_conversation/list_row.ui");
-        outer_grid = (Grid) builder.get_object("outer_grid");
+        outer_box = (Box) builder.get_object("outer_box");
         picture = (AvatarPicture) builder.get_object("picture");
         name_label = (Label) builder.get_object("name_label");
+        status_dot = (Image) builder.get_object("status_dot");
         via_label = (Label) builder.get_object("via_label");
 
+
         this.layout_manager = new BinLayout();
-        outer_grid.set_parent(this);
+        outer_box.set_parent(this);
     }
 
     public ListRow() {}
 
+    private void set_status_dot(StreamInteractor stream_interactor, Jid jid, Account account){
+        Gee.List<Jid>? full_jids = stream_interactor.get_module(PresenceManager.IDENTITY).get_full_jids(jid, account);
+        string presence_str = null;
+        if (full_jids != null){
+            int devices = 0;
+            string newline_if_more_devices;
```

Comment:
This variable isn't used anymore

Reply:
Yeah I noticed that, oops! I did have an amend that I'll force push today